media/css/custom.css

```diff
@@ -894,3 +894,34 @@ summary {
 .struct-elem:hover {
   background: #0044cc;
 }
+
+.relax-item {
+  width: 200px;
+  height: 200px;
+  background: lavenderblush;
+  float: left;
+  margin-left: 20px;
+  margin-top: 10px;
+}
+
+.relax-img {
+  width: 100%;
+  height: 150px;
+  background: silver;
+}
+
+.relax-content {
+  width: 890px;
+}
+
+.relax-description {
+  height: 50px;
+  width: 100%;
+}
+
+.relax-description a {
+  display: block;
+  margin-top: 8px;
+  margin-left: 10px;
+  text-align: left;
+}
```

Comment:
Tweak setting of your editor of choice to get rid of this [No new line at end of file](https://robots.thoughtbot.com/no-newline-at-end-of-file) message.